constraints.go

```diff
@@ -42,7 +42,7 @@ func init() {
 		cvRegex))
 
 	constraintRangeRegex = regexp.MustCompile(fmt.Sprintf(
-		`\s*(%s)\s*-\s*(%s)\s*`,
+		`\s*(%s)\s* - \s*(%s)\s*`,
```

Comment:
My only nit would be that `\s* ` and ` \s*` could be replaced with `\s+`. The `*` is for zero or more while the `+` is for one or more.